config/routes.rb

```diff
@@ -1,8 +1,11 @@
 Rails.application.routes.draw do
   root "static_pages#home"
-  get "help", to: "static_pages#help"
-  get "about", to: "static_pages#about"
-  get "contact", to: "static_pages#contact"
-  get "signup", to: "users#new"
-  resources :users, only:[:show, :create]
+  get "/help", to: "static_pages#help"
+  get "/about", to: "static_pages#about"
+  get "/contact", to: "static_pages#contact"
+  get "/signup", to: "users#new"
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
+  resources :users, only:[:show, :create, :destroy]
```

Comment:
đọc về ruby percent notation em nhé  
```suggestion
  resources :users, only: %i(show create destroy)
```